content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -358,6 +362,10 @@ For example, the total amount of free "conventional" memory in the group can be
 
 The line `"systemReserved":3221225472` indicates that the administrator of this node reserved `3221225472` bytes (i.e. `3Gi`) to serve kubelet and system processes at NUMA node `0`, by using `--reserved-memory` flag.
 
+### Resource API
+
+By employing [Resource API](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/), the information about reserved memory for each container can be retrieved through `ContainerMemory` message.   

```

Comment:
I don't understand what point we're making here. If you can write more detail here, a SIG Docs reviewer can help frame that more concisely.

Reply:
The device plugin (resource management) team decided that their API will be called "resource API". The clarification here makes sense.

Reply:
`ContainerMemory` is a new protobuf definition added to keep data about the memory reserved for containers.
Here is initial work on "Resource API" and definition of `protobufs`:
- https://github.com/kubernetes/enhancements/pull/1884/files
- https://github.com/kubernetes/enhancements/pull/2404/commits/9e0402978eed0e5611261fb89b98afaa851e7d0a
Please advise if could be referred in more suitable way.